ui-tests/src/test/java/org/openmrs/reference/DeleteFormTest.java

```diff
@@ -50,7 +51,9 @@ public void deleteFormTest() throws Exception {
 
     @After
     public void tearDown() throws Exception {
+	if(headerPage !=null) {
```

Comment:
Any reason for adding this

Reply:
This handles nullpointer exception in headerPage